server/db.js

```diff
@@ -29,4 +39,5 @@ const getLoc = (info, callback) => {
 };
 
 module.exports.getProp = getProp;
+module.exports.getAllProp = getAllProp;
 module.exports.getLoc = getLoc;
```

Comment:
I would say  that maybe have a more descriptive names in regards to what your methods do.